api/src/main/scala/dcos/metronome/api/v1/controllers/JobRunController.scala

```diff
@@ -65,13 +67,16 @@ class JobRunController(
   }
 
   def triggerJob(id: JobId) = measured {
-    AuthorizedAction.async { implicit request =>
+    AuthorizedAction.async(validate.optionalJson[JobRunSpecOverrides]) { implicit request =>
+      val overrides = request.body
+      JobRunController.log.info(s"overrides: $overrides")
```

Comment:
do we want to log this in production? I mean we don't log other requests... 🤔

Reply:
removed